src/MCPServer/lib/server/queues.py

```diff
@@ -196,20 +196,21 @@ def stop(self):
         """
         self.shutdown_event.set()
 
-    def _package_completed_callback(self, package, future):
-        """Called by an executor on completion of a package. Marks the package
-        as inactive and schedules a new package.
+    def _package_completed_callback(self, package, link_id, future):
+        """Called by an executor on completion of a job.
```

Comment:
Nit: maybe move this to the body of the commend and move the first line of the comment here.